dom/base/nsContentUtils.cpp

```diff
@@ -7760,6 +7760,13 @@ nsContentUtils::SendMouseEvent(nsCOMPtr<nsIPresShell> aPresShell,
     msg = eMouseUp;
   } else if (aType.EqualsLiteral("mousemove")) {
     msg = eMouseMove;
+
+    // If need to draw mouse cursor, set point then force composition
+    if(widget->GetDrawMouse()) {
+      ScreenIntPoint point(floor(aX+0.5), floor(aY+0.5));
+      widget->SetScreenIntPoint(point);
+      widget->ForceComposition();
```

Comment:
Yes, because nsContentUtils::SendMouseEvent is the implementation of nsIDOMWindowUstils::SendMouseEvent. I want promise if WebApp or others use nsIDOMWindowUtils won't get a surprising mouse cursor on screen.